HeterogeneousCore/AlpakaTest/plugins/TestAlpakaAnalyzer.cc

```diff
@@ -128,7 +128,7 @@ class TestAlpakaAnalyzer : public edm::global::EDAnalyzer<> {
     assert(view.r() == 1.);
     for (int32_t i = 0; i < view.metadata().size(); ++i) {
       auto vi = view[i];
-      assert(vi.x() == 0.);
+      //assert(vi.x() == 0.);
```

Comment:
this and a few more changes in HeterogeneousCore/AlpakaTest don't clearly belong to sistrip developments.
related commits should probably be dropped from history